### PR TITLE
Markdown Fixes to 7.5

### PR DESCRIPTION
**Pull request by ArieHein**
# PR Summary
Fixing markdown warnings to align with the markdown rules per https://github.com/MicrosoftDocs/PowerShell-Docs/blob/main/.markdownlint.yaml

Specifically:
MD022 (with exception to SYNOPSIS)   saved you about 300 changes, *thanks for the heads up @sdwheeler !*
MD012

One exception is in the PSReadLine lines 51-52 that triggered MD024 as its a duplicate of lines 42-43

Next batch containing some other MD rules for 7.5 and content coming up as well this week.

- [x] **Descriptive Title:** This PR's title is a synopsis of the changes it proposes.
- [x] **Summary:** This PR's summary describes the scope and intent of the change.
- [x] **Contributor's Guide:** I have read the [contributors guide][contrib].
- [x] **Style:** This PR adheres to the [style guide][style].

[contrib]: https://learn.microsoft.com/powershell/scripting/community/contributing/overview
[style]: https://learn.microsoft.com/powershell/scripting/community/contributing/powershell-style-guide

### CODE REVIEW

**Comment by ArieHein** [comment]
Also @sdwheeler - any preferences on fixing things like for example about_ANSI_terminals.md  in lines 223 that triggers a spelling error as it has "pwsh -noprofile -command 'Get-ChildIitem' > out.txt"  when it should be  'Get-ChildItem' 
So making sure the casing is correct in all code examples ? There are quite a few of them all over that just cause my old style PS OCD to kick in :)

Also, are you accepting adding more words to the spelling dictionaries ?

**Comment by sdwheeler** [comment]
@ArieHein check your Discord account.

File: reference/7.5/Microsoft.PowerShell.Core/Get-Module.md [hunk 1]
```diff
@@ -549,6 +549,7 @@ The `Get-Module` cmdlet checks **CompatiblePSEditions** property of **PSModuleIn
 specified value and returns only those modules that have it set.
 
 > [!NOTE]
+>
```
Comment (sdwheeler) [inline reference/7.5/Microsoft.PowerShell.Core/Get-Module.md:552]: Let's not put a blank line in the alert blocks like this.
```suggestion
```

File: reference/7.5/PSReadLine/PSReadLine.md [hunk 2]
```diff
@@ -40,22 +40,25 @@ These articles document version 2.3.5 of PSReadLine.
...
 ### [Get-PSReadLineKeyHandler](Get-PSReadLineKeyHandler.md)
+
 Gets the key bindings for the PSReadLine module.
 
 ### [Get-PSReadLineOption](Get-PSReadLineOption.md)
-Gets values for the options that can be configured.
 
-### [PSConsoleHostReadLine](PSConsoleHostReadLine.md)
-This function is the main entry point for PSReadLine.
+Gets values for the options that can be configured.
```
Comment (sdwheeler) [inline reference/7.5/PSReadLine/PSReadLine.md:52]: Why was this removed? We want to keep this here.

**Review by sdwheeler — CHANGES_REQUESTED** [review]
See my suggestions.

File: reference/7.5/Microsoft.PowerShell.Core/Get-Module.md (hunk 1 above)
Comment (sdwheeler) [reply reference/7.5/Microsoft.PowerShell.Core/Get-Module.md:552]: Yeah, treat this as another exception. Remove line 552.

File: reference/7.5/PSReadLine/PSReadLine.md (hunk 2 above)
Comment (sdwheeler) [reply reference/7.5/PSReadLine/PSReadLine.md:52]: remove the 42-43 and bring back the 51-52

**Review by sdwheeler — APPROVED** [review]
